### PR TITLE
Feature/cloud services

### PR DESCRIPTION
**Pull request by amitschendel**

### CODE REVIEW

File: pkg/dnsmanager/dns_manager.go [hunk 1]
```diff
@@ -91,3 +109,58 @@ func (dm *DNSManager) ResolveIPAddress(ipAddr string) (string, bool) {
...
+
+	gcpDomains := []string{
+		"googleapis.com.",
+		"appspot.com.",
+		"cloudfunctions.net.",
+		"run.app.",
+	}
+
+	// Check for other common cloud providers
+	otherCloudDomains := []string{
```
[the remark this answers is not in the document]
Comment (matthyx) [reply pkg/dnsmanager/dns_manager.go:145]: not sure where to find them...

[the remark this answers is not in the document]
Comment (matthyx) [reply pkg/dnsmanager/dns_manager.go:145]: I think there are some domains in *.ovh.net

File: pkg/exporters/http_exporter.go [hunk 2]
```diff
@@ -58,186 +73,231 @@ type HTTPAlertsListSpec struct {
...
 	if config.Method == "" {
-		config.Method = "POST"
+		config.Method = defaultMethod
 	} else if config.Method != "POST" && config.Method != "PUT" {
 		return fmt.Errorf("method must be POST or PUT")
 	}
+
 	if config.TimeoutSeconds == 0 {
-		config.TimeoutSeconds = 5
+		config.TimeoutSeconds = int(defaultTimeout.Seconds())
```
Comment (matthyx) [inline pkg/exporters/http_exporter.go:106]: looks weird to have a duration just to extract an int... tell Claude I would have preferred `defaultTimeoutSeconds` here

**Review by matthyx — COMMENTED** [review]
overall nice refactoring of the `http_exporter`, let's see if we can find OVH domains before merging

File: pkg/exporters/http_exporter.go (hunk 2 above)
Comment (amitschendel) [reply pkg/exporters/http_exporter.go:106]: I actually like this, it's readable.

File: pkg/dnsmanager/dns_manager.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (amitschendel) [reply pkg/dnsmanager/dns_manager.go:145]: Currently we only care about aws anyway.

File: pkg/exporters/http_exporter.go (hunk 2 above)
Comment (matthyx) [reply pkg/exporters/http_exporter.go:106]: yeah... and it's probably too late to change the config timeoutSeconds into timeoutDuration :(

**Review by matthyx — DISMISSED** [review]

**Pull request merged**